src/engine/controls/cuecontrol.cpp

```diff
@@ -1990,6 +1990,24 @@ bool CueControl::isTrackAtIntroCue() {
             1.0f);
 }
 
+bool CueControl::inIntro() const {
+    if (!m_pIntroStartEnabled->toBool() || !m_pIntroEndEnabled->toBool()) {
+        return false;
+    }
+
+    const double pos = getSampleOfTrack().current;
```

Comment:
Please add units to variable names (e. g. `framePosition` or `positionSamples` or something like that)